examples/sample_pipeline/run.sh

```diff
@@ -27,6 +27,8 @@ trap cleanup EXIT
 # Bind local data directory to pipeline
 data_dir=$(readlink -f "data")
 
+export FONDANT_VERSION=$GIT_HASH
```

Comment:
I think you are not using the `GIT_HASH`.

Reply:
The `FONDANT_VERSION` environment variable is used in `pipeline.py` (you commented on it :slightly_smiling_face:)